node-notes-cli/remove.js

```diff
@@ -0,0 +1,25 @@
+import { readFileSync } from 'node:fs';
+import { readFile, writeFile } from 'node:fs/promises';
+export default remove;
+
+const data = readFileSync('data.json', 'utf-8');
+const dataParse = JSON.parse(data);
+const number = process.argv[3];
+
+function remove() {
+  readFile('data.json')
+    .then((data) => {
+      for (const property in dataParse.notes) {
```

Comment:
This loop isn't necessary. You can just `delete dataParse.notes[number];` because if you try to delete a property that doesn't exist, `delete` does nothing.